lib/chef/knife/ssh.rb

```diff
@@ -123,6 +123,11 @@ class Ssh < Knife
         :boolean => true,
         :proc => Proc.new { :raise }
 
+      option :skip_on_duplicated_fqdns,
+        :long => "--skip-on-duplicated-fqdns",
+        :description => "Skip exits even if FQDNs are duplicated",
+        :boolean => true
```

Comment:
this is still the incorrect default here.  it needs to be off by default.  it should also be expanded to support different behaviors:  `:ignore`, `:warn`, `:fatal`, `:skip`

Reply:
I got it.